docs/cimpl6ToolingReference.md

```diff
@@ -281,6 +270,24 @@ Here is an example of provenanceInformation:
         "copyright": "Copyright (c) The Example Organization <http://example.org>"
     }
 ```
+#### Dependencies
+
+The [HL7 FHIR U.S. Core](https://www.hl7.org/fhir/us/core/) IG is included by default as a dependency for all IGs created using CIMPL.
+
+To add other dependency IGs, they must be created using CIMPL, and the Configuration file `"dependencies"` parameter must be valued. This information is copied into the [HL7 FHIR IG Publisher control file dependencyList](https://wiki.hl7.org/index.php?title=IG_Publisher_Documentation#DependencyList). Here is an example:
+
+````
+"dependencies": [
+        {
+        "package": "hl7.fhir.us.mcode",
+        "version": "current",
+        "name": "mcode",
+        "location": "http://hl7.org/fhir/us/mcode/2019Sep//",
+        "fileLocation": "../../Desktop/mcode"
+        }
+]
+````
+> **Note**:The dependent IG JSON definitions must be downloaded into the folder referenced by `fileLocation`. 
```

Comment:
I don't think we need to say that the other IG "must be created using CIMPL", for example, people have used this feature to depend upon the VHDir IG http://hl7.org/fhir/uv/vhdir/2018Jan/index.html, which was not made using CIMPL. I also think it is important to mention that once a dependency is included, the user can map their profiles to profiles from that dependency IG. Also in "location" we could remove the trailing "//".